SetupDataPkg/Library/ConfigVariableListLib/ConfigVariableListLib.c

```diff
@@ -185,13 +233,24 @@ ConvertVariableEntryToVariableList (
     goto Exit;
   }
 
-  NameSize = StrnSizeS (VariableEntry->Name, CONF_VAR_NAME_LEN);
+  Status = SafeUintnToUint32 (StrnSizeS (VariableEntry->Name, CONF_VAR_NAME_LEN), &NameSize);
+  if (EFI_ERROR (Status)) {
+    // overflowed...
+    DEBUG ((DEBUG_ERROR, "%a VarListEntry name %a overflowed, too long a name!\n", __FUNCTION__, VariableEntry->Name));
+    goto Exit;

```

Comment:
nit: Given this is bound to `CONF_VAR_NAME_LEN`, i think we do not have to use the safe int here.

But if it really overflow, we probably do not want to print the bogus name to run into illegal access.

Reply:
Agreed, wasn't paying attention and got overzealous with the SafeIntLib :). I'll remove this usage.